rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,725 @@
+        the interpolation to use if transforming points not on the array's discrete grid.
+        Values could be:
+        <ul>
+            <li><code>linear</code> (default)</li>
+            <li><code>nearest</code></li>
+            <li><code>cubic</code></li>
+        </ul></dd>
+</dl>
+
+
+For both `coordinates` and `displacements`, the array data at referred to by `path` MUST define coordinate system and coordinate transform metadata:
+
+* Every axis name in the `coordinateTransform`'s `input` MUST appear in the coordinate system
+* The array dimension corresponding to the `coordinate` or `displacement` axis MUST have length equal to the number of dimensions of the `coordinateTransform` `output`
+* If the input coordinate system `N` axes, then the array data at `path` MUST have `(N + 1)` dimensions.
+* SHOULD have a `name` identical to the `name` of the corresponding `coordinateTransform`.
+
+For `coordinates`:
+
+* `coordinateSystem` metadata MUST have exactly one axis with `"type" : "coordinate"`
+* the shape of the array along the "coordinate" axis must be exactly `N`
+
+For `displacements`:
+
+* `coordinateSystem` metadata MUST have exactly one axis with `"type" : "displacement"`
+* the shape of the array along the "displacement" axis must be exactly `N`
+* `input` and `output` MUST have an equal number of dimensions.
+
+
+#### <a name=trafo-byDimension>byDimension</a>
+
+`byDimension` transformations build a high dimensional transformation using lower dimensional transformations
+on subsets of dimensions.
+
+<dl>
+  <dt><strong>transformations</strong></dt>
+  <dd>  A list of transformations, each of which applies to a (non-strict) subset of input and output dimensions (axes). 
+        The values of <code>input</code> and <code>output</code> fields MUST be an array of strings.
+        Every axis name in <code>input</code> MUST correspond to a name of some axis in this parent object's <code>input</code> coordinate system.
+        Every axis name in the parent byDimension's <code>output</code> MUST appear in exactly one of its child transformations' <code>output</code>.
+        </dd>
+</dl>
+
+
+#### <a name="bijection">bijection</a>
+
+A bijection transformation is an invertible transformation in which both the `forward` and `inverse` transformations
+are explicitly defined. Each direction SHOULD be a transformation type that is not closed-form invertible.
+Its' input and output spaces MUST have equal dimension. The input and output dimensions for the both the forward
+and inverse transformations MUST match bijection's input and output space dimensions.
+
+`input` and `output` fields MAY be omitted for the `forward` and `inverse` transformations, in which case
+the `forward` transformation's `input` and `output` are understood to match the bijection's, and the `inverse`
+transformation's `input` (`output`) matches the bijection's `output` (`input`), see the example below.
```

Comment:
Should clarify whether the bijection's `input` and `output` must be specified (like for sequence).